tests/cpython-tests/Makefile

```diff
@@ -2,18 +2,23 @@ TOP=$(abspath ../..)
 include $(TOP)/defs.mak
 
 APPBUILDER=$(TOP)/scripts/appbuilder
-HEAP_SIZE="4G"
-OPTS += --memory-size $(HEAP_SIZE) --nobrk
+
+OPTS = --app-config-path config.json
 ifdef STRACE
 OPTS += --strace
 endif
+
+#Linux target ignores config.json, add equivalent command line options as a workaround
+HEAP_SIZE="4G"
+OPTS += --memory-size $(HEAP_SIZE) --nobrk
```

Comment:
This can be wrapped in a `ifeq ($TARGET, linux)`.

Reply:
In fact, Linux target does process config.json, but have slightly different logic than sgx target. Luckily, for this specific config.json, the difference in the logic does not have negative impact. Removed the command line options in the latest update to the PR.